### PR TITLE
fix: moved enable condition properties out of style

### PR DESCRIPTION
**Pull request by eliseeborn**

### CODE REVIEW

File: src/utils/style-formatter.js [hunk 1]
```diff
@@ -42,18 +40,18 @@ export default {
...
+    disabled && (styles += formatProperty('opacity', 0.4));
 
     if (style.background && style.background.isUsed) {
       let bgUrl = style.background.url.qStaticContentUrl.qUrl;
       bgUrl.replace(/^\.\.\//i, '/');
       bgUrl = bgUrl.replace(/"/g, '\\"');
       bgUrl = bgUrl.replace(/'/g, "\\'");
       styles += formatProperty('background-image', `url('${bgUrl}')`);
-      styles += formatProperty('background-size', backgroundSize[style.background.size] || backgroundSize.auto);
+      styles += formatProperty('background-size', backgroundSize[style.background.size]);
```
Comment (haxxmaxx) [inline src/utils/style-formatter.js]: I've removed the default values, so we need to keep this or put the default values back.

Comment (eliseeborn) [reply src/utils/style-formatter.js]: ah, I got confused then - I'll put them back. :)

File: src/utils/__tests__/style-formatter.spec.js [hunk 2]
```diff
@@ -96,25 +97,19 @@ describe('style-formatter', () => {
...
     });
 
     it('should have set opacity for disabled button', () => {
-      style = {
-        useEnabledCondition: true,
-        enabledCondition: 0
-      };
-      const formattedStyle = styleFormatter.getStyles(style, Theme);
+      disabled = true;
+      const formattedStyle = styleFormatter.getStyles(style, disabled, Theme);
```
Comment (niekvanstaveren) [inline src/utils/__tests__/style-formatter.spec.js]: You could set true inside the function here as well. Then you don't have to reset disabled in the next test

Comment (eliseeborn) [reply src/utils/__tests__/style-formatter.spec.js]: like this?
`const formattedStyle = styleFormatter.getStyles(style, true, Theme);`

Comment (niekvanstaveren) [reply src/utils/__tests__/style-formatter.spec.js]: exactly! In that way you don't change the default values and start every unit test in the same way. Another way would be to have a beforeEach function, but that seems a bit much for this single test

**Pull request merged**